### PR TITLE
Log network errors and their corresponding query.

### PR DESCRIPTION
**Pull request by mcstover**
Open to other ideas here. With regard to the ECONNRESET errors, there is a chance that our queries are malformed and this will help us inspect those and to determine if any particular query is the culprit.

### CODE REVIEW

**Comment by mcstover** [comment]
@emuvente going to throw this into test and maybe stage and see if it yields anything useful

File: src/api/NetworkErrorLink.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	return onError(({
+		networkError,
+		operation,
+		forward
+	}) => {
+		if (networkError) {
+			logFormatter(operation, 'error');
+			logFormatter(networkError, 'error');
+		}
+		forward();
```
Comment (emuvente) [inline src/api/NetworkErrorLink.js:14]: Looking at examples, calling this doesn't appear necessary

Comment (mcstover) [reply src/api/NetworkErrorLink.js:14]: I thought so too! Especially looking at the code for the error link. However, when I initially tried without it it that made this a terminating link and halted the flow to other links...

Comment (emuvente) [reply src/api/NetworkErrorLink.js:14]: aha

**Pull request merged**